`BindableMetadata.g.cs` contains types which do not exist

### Current behavior

Getting:

```
BindableMetadata.g.cs(27360,96,27360,114): error CS0234: The type or namespace name 'SwipeRefreshLayout' does
not exist in the namespace 'AndroidX' (are you missing an assembly reference?)
```

Without `SwipeRefreshLayout` NuGet being included.

### Expected behavior

Should not happen.

### How to reproduce it (as minimally and precisely as possible)

- Clone https://github.com/MartinZikmund/SmsTicket/tree/uno-bug-repro-bindable-metadata
- Try to build `SmsTicket.Mobile`

### Workaround

_No response_

### Works on UWP/WinUI

None

### Environment

_No response_

### NuGet package version(s)

_No response_

### Affected platforms

_No response_

### IDE

_No response_

### IDE version

_No response_

### Relevant plugins

_No response_

### Anything else we need to know?

_No response_

--- a/build/nuget/Uno.WinUI.nuspec
+++ b/build/nuget/Uno.WinUI.nuspec
@@ -31,6 +31,7 @@
 				<dependency id="Xamarin.AndroidX.RecyclerView" version="1.2.1.3" />
 				<dependency id="Xamarin.AndroidX.Activity" version="1.3.1.2" />
 				<dependency id="Xamarin.AndroidX.Browser"  version="1.0.0" />
+				<dependency id="Xamarin.AndroidX.SwipeRefreshLayout"  version="1.1.0.18" />
 				<dependency id="Uno.Foundation.Logging" version="0.0.0" />
 				<dependency id="Uno.Diagnostics.Eventing" version="2.0.1" />
 				<dependency id="Uno.Fonts.Fluent" version="2.3.0" />
@@ -63,6 +64,7 @@
 				<dependency id="Xamarin.AndroidX.RecyclerView" version="1.2.1.3" />
 				<dependency id="Xamarin.AndroidX.Activity" version="1.3.1.2" />
 				<dependency id="Xamarin.AndroidX.Browser"  version="1.0.0" />
+				<dependency id="Xamarin.AndroidX.SwipeRefreshLayout"  version="1.1.0.18" />
 				<dependency id="Uno.Foundation.Logging" version="0.0.0" />
 				<dependency id="Uno.Diagnostics.Eventing" version="2.0.1" />
 				<dependency id="Uno.Fonts.Fluent" version="2.3.0" />

--- a/src/Directory.Build.targets
+++ b/src/Directory.Build.targets
@@ -131,6 +131,7 @@
 		<PackageReference Update="Xamarin.AndroidX.RecyclerView" Version="1.2.1.3" />
 		<PackageReference Update="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.3.1.3" />
 		<PackageReference Update="Xamarin.AndroidX.Fragment" Version="1.3.6.3" />
+		<PackageReference Update="Xamarin.AndroidX.SwipeRefreshLayout" Version="1.1.0.18" />
 	</ItemGroup>
 
 	<ItemGroup Condition=" '$(TargetFramework)' == 'net7.0-android'">
@@ -141,6 +142,7 @@
 		<PackageReference Update="Xamarin.AndroidX.RecyclerView" Version="1.2.1.3" />
 		<PackageReference Update="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.3.1.3" />
 		<PackageReference Update="Xamarin.AndroidX.Fragment" Version="1.3.6.3" />
+		<PackageReference Update="Xamarin.AndroidX.SwipeRefreshLayout" Version="1.1.0.18" />
 	</ItemGroup>
 
 	<Target Name="ValidateSolutionPath" BeforeTargets="Build">

--- a/src/Uno.UI/UI/Xaml/Controls/TextBox/TextBoxView.Android.cs
+++ b/src/Uno.UI/UI/Xaml/Controls/TextBox/TextBoxView.Android.cs
@@ -216,7 +216,7 @@ namespace Microsoft.UI.Xaml.Controls
 						{
 							var colorFilter = BlendModeColorFilterCompat.CreateBlendModeColorFilterCompat(
 								(Android.Graphics.Color)color,
-								BlendModeCompat.SrcAtop);
+								BlendModeCompat.SrcAtop)!;
 							drawable?.SetColorFilter(colorFilter);
 						}
 					}

--- a/src/Uno.UI/UI/Xaml/Controls/TextBox/TextBoxView.Android.cs
+++ b/src/Uno.UI/UI/Xaml/Controls/TextBox/TextBoxView.Android.cs
@@ -216,7 +216,7 @@ namespace Microsoft.UI.Xaml.Controls
 						{
 							var colorFilter = BlendModeColorFilterCompat.CreateBlendModeColorFilterCompat(
 								(Android.Graphics.Color)color,
-								BlendModeCompat.SrcAtop)!;
+								BlendModeCompat.SrcAtop!);
 							drawable?.SetColorFilter(colorFilter);
 						}
 					}

--- a/src/Uno.UI/Uno.UI.netcoremobile.csproj
+++ b/src/Uno.UI/Uno.UI.netcoremobile.csproj
@@ -66,6 +66,7 @@
 		<PackageReference Include="Xamarin.AndroidX.AppCompat" />
 		<PackageReference Include="Xamarin.AndroidX.RecyclerView" />
 		<PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" />
+		<PackageReference Include="Xamarin.AndroidX.SwipeRefreshLayout" />
 	</ItemGroup>
 
 	<!-- This condition is **explicitly** for net7.0 -->

